Automattic/sensei
# Create tables-based versions of student progress and quiz data when saving of their comment based versions

Resolves #7078 

## Proposed Changes
* Create course/lesson/quiz tables based progress if not exist while saving their comments based versions.
* Create quiz submissions/answers/grades if not exist while saving their comments based versions.

## Testing Instructions

1. Disable the feature flag, take a course and pass a quiz.
2. Enable the feature flag (`tables_based_progress`).
3. Check you don't have submission, answers and grades for the submission from the first step in DB.
4. Go to Sensei LMS -> Grades, grade and save the submission.
5. Make sure you have all needed tables based versions.

## Pre-Merge Checklist
